client/src/preview.css

```diff
@@ -0,0 +1,89 @@
+.layoutContainer {
+  margin: 80px 0 0 0;
+  height: 47vh;
+  display: flex;
+  align-items: flex-start;
+}
+
+.primaryImg {
+  height: 47vh;
+  width: 67%;
+  overflow: hidden;
+  outline: 1px solid rgb(72, 72, 72);
+  border-left: 1px solid rgb(72, 72, 72);
+  border-right: 1px solid rgb(72, 72, 72);
+}
+
+.img {
+  max-width: 100%;
+  cursor: pointer;
+}
+
+.secondaryImgsContainer {
+  height: 47vh;
+  width: 33%;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+}
+
+.secondaryImg {
+  height: 50%;
+  overflow: hidden;
+  outline: 1px solid rgb(72, 72, 72);
+  border-left: 1px solid rgb(72, 72, 72);
+  border-right: 1px solid rgb(72, 72, 72);
+}
+
+.listingActions {
+  position: absolute;
+  top: 24px;
+  right: 24px;
+}
+
+.shareListing {
+  margin-right: 18px;
+  cursor: pointer;
+  outline: none;
+  color: #484848;
```

Comment:
The button logic for share listing / save listing / view photos is really repetitive... ideally I'd find a way to not repeat